dali/kernels/signal/dct/dct_gpu.h

```diff
@@ -77,12 +77,14 @@ class DLL_PUBLIC Dct1DGpu {
 
   DLL_PUBLIC KernelRequirements Setup(KernelContext &context,
                                       const InListGPU<InputType> &in,
-                                      span<const DctArgs> args, int axis);
+                                      span<const DctArgs> args, int axis,
+                                      span<const float> lifter_coeffs);
```

Comment:
If it's not used in Setup, don't add it.

Reply:
done